### PR TITLE
Add the other CSharp Project

### PR DESCRIPTION
**Pull request by Skul1CrowN**
coffee-to-code with csharp

### CODE REVIEW

**Review by narze — CHANGES_REQUESTED** [review]
Thanks for contribution! 
By the way you can submit only single source code file 😅 Can you remove all other files?

File: C#/Skul1CrowN/Program.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    public static void Main()
+    {
+        string? input = Console.ReadLine();
+        if(input == null)
+        {
+            Console.WriteLine("Something Went Wrong");
+        }
+        else
+        {
+            input = Regex.Replace(input,"coffee","code", RegexOptions.IgnoreCase);
```
Comment (narze) [inline C#/Skul1CrowN/Program.cs:14]: Only this file is ok

Comment (Skul1CrowN) [reply C#/Skul1CrowN/Program.cs:14]: Ok, I will resubmit only that file.

Comment (Skul1CrowN) [reply C#/Skul1CrowN/Program.cs:14]: I removed the other unnecessary files.

**Review by narze — APPROVED** [review]
👍🏻

**Pull request merged**